packages/lwc-language-server/src/javascript/__tests__/compiler.test.ts

```diff
@@ -215,68 +107,96 @@ it('transform returns javascript metadata', async () => {
     const filepath = path.join('src', 'javascript', '__tests__', 'fixtures', 'metadata.js');
```

Comment:
We significantly augmented the fixture used in this test. Every variety of decorator that we could think of was added. That is reflected in the changes below.